### ISSUE TITLE
Who are the maintainers?

### ISSUE THREAD

**Issue by simonepri** [indexzero/ps-tree#30]
Following this old issue:
https://github.com/indexzero/ps-tree/issues/10

Who are the current maintainers of this package?
For what I saw:
@indexzero is the author
@nelsonic is one of the maintainers 
Am I missing someone?

> Can we add more contributors to this repo?
https://github.com/indexzero/ps-tree/issues/18#issuecomment-373294868

cc @zixia @soyuka

**Comment by soyuka** [comment]
We maybe could release a community fork? I've a pending fix on @zixia 's fork also haha

**Comment by simonepri** [comment]
Let's see what @nelsonic says about that.
@soyuka are you referring to this https://github.com/zixia/ps-tree/pull/1?
Can you create the PR on this repo too?

**Comment by soyuka** [comment]
> Can you create the PR on this repo too?

It's based on @zixia ´s work on the repository so I guess we need to merge his changes first :|

**Comment by simonepri** [comment]
Oh right!
So let's make a single PR with both changes.

**Comment by zixia** [comment]
@soyuka Let me merge your change in my PR, then I believe my PR at here will include your PR....

:-/

**Comment by simonepri** [comment]
@zixia can you open another PR removing the addition of appveyor?
We have already https://github.com/indexzero/ps-tree/pull/29 for that.

**Comment by zixia** [comment]
@simonepri Ok no problem.

I had just modified my PR and remove the README&Appveyor changes.

Please let me know if you think I should modify other parts as well.

**Comment by simonepri** [comment]
Thank you! I've commentend under the PR.

**Comment by simonepri** [comment]
@indexzero See also this
https://github.com/indexzero/ps-tree/pull/28#issuecomment-373663239

**Comment by soyuka** [comment]
FTR merged #17 #24 #27 and #28 to https://github.com/soyuka/ps-tree

**Comment by simonepri** [comment]
@soyuka Lets wait @nelsonic before proceed please.

I don't see any advantage of working on a community fork if we can push on npm from this one.

**Comment by soyuka** [comment]
@simonepri  I don't want to bother anyone ;) I'm just using this in `pidusage-tree`, and therefore will use my fork for the time being :). Obviously if things gets handled here I'll use the original version.

https://twitter.com/andrestaltz/status/961600427832872961

**Comment by simonepri** [comment]
@nelsonic @indexzero news?

**Comment by soyuka** [comment]
@simonepri pinging them is useless just let them answer if they want to I'm sure they've been notified by now ;). Note that my last contributions to this were almost 1 year ago and there has been no answer yet.

**Comment by simonepri** [comment]
Actually nelsonic have already merged some of my PRs some days ago:
https://github.com/indexzero/ps-tree/pull/26
https://github.com/indexzero/ps-tree/pull/25

**Comment by zixia** [comment]
But it seems that @nelsonic only merge _Badges_...

**Comment by zixia** [comment]
@simonepri I saw your `pidtree` npm, and thanks for adding me to the collaborators.

It seems we have to do a hard fork for this module finally...

**Comment by indexzero** [comment]
@huan @simonepri if you'd like to discuss merging back into a `ps-tree@2.0.0` API please let me know. My apologies for not responding sooner. This was on my to-do list, but kept getting pushed back.

**Comment by indexzero** [comment]
Updating the title of this issue to reflect the current discussion.

**Comment by simonepri** [comment]
@indexzero `pidtree` is a from scratch re-work of this package not 100% sure that having it merged back here is the right choice. @soyuka

**Comment by indexzero** [comment]
@simonepri exactly! And it has an improved algorithm that is more correct. 

This module is downloaded about 5,000,000 times a month. Wouldn't it be great if all those folks got a better quality module?

By releasing it as 2.0.0 it gives folks an upgrade path that is opt-in as well.

If you are interested I would add you as an owner and a maintainer on nom. Let me know! ^_^

**Comment by huan** [comment]
@indexzero it's great to see you wake up and come back to us! :D

**Comment by simonepri** [comment]
@indexzero Your point is right.
I'm still not 100% happy with the implementation of `pidtree` and I would like to do some rework before starting to think about merging it back here.

My idea is to create separate packages for each platform and then to have `pidtree` or `ps-tree` to be just a wrapper for the right implementation.
Also, I wold like to add a more sophisticated cross-platform testing though docker.

The ETA of this would be at least a month, since I'm super busy right now.

**Comment by indexzero** [comment]
@simonepri that makes sense. Given your past excellent work on PR #24, #27, and #29 I've added you as a collaborator to this project. When you accept that invite I am happy to make you a full Administrator.

Once you've reached a place you're happy with from an implementation perspective please let me know and I will add you as an owner on `npm`.

**Comment by indexzero** [comment]
So it turns out that for personal repositories they only the person they are associated with can be administrators. I am going to move this over to the `foreverjs` org since it's heavily depended on by that package –– unless there are other thoughts of course.
